bin/retrieve-documentation

```diff
@@ -5,10 +5,14 @@ cd "$(dirname "$0")/../"
 # Remove previous documentation
 
 rm -fr src/pages/docs
+rm -fr src/pages/images
 
 # Checkout documentation
 
-git clone https://github.com/api-platform/docs.git src/pages/docs
+git clone -b `git remote show git@github.com:api-platform/docs.git | grep -e"Branche HEAD \| HEAD branch"| cut -d ":" -f 2` --single-branch https://github.com/api-platform/docs.git src/pages/docs
```

Comment:
Hmm... French? :confused: So the output is dependent on the user's system language?

Reply:
Add ˋLANG=C` before the command to force the lang to be default one.